src/hooks/useScrollBlock.js

```diff
@@ -0,0 +1,51 @@
+import { useRef } from "react";
+
+const safeDocument = document;
+
+/**
+ * Usage:
+ * const [blockScroll, allowScroll] = useScrollBlock();
+ */
+export const useScrollBlock = () => {
```

Comment:
where is this code from?

Reply:
Sorry I forgot to add the source in a comment. It was from a Github gist. Is fixed now

Reply:
Ok